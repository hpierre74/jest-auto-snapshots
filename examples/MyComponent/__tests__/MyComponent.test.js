import snap from 'jest-auto-snapshots';
import MyComponent from '../MyComponent';

snap(MyComponent);
