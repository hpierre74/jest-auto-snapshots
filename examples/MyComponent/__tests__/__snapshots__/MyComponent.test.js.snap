// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest-auto-snapshots > MyComponent Matches snapshot when boolean prop "booleanProp" is set to: "false": when boolean prop "booleanProp" is set to: "false" 1`] = `
<div>
  jest-auto-snapshots String Fixture
  <NodeFixture />
</div>
`;

exports[`jest-auto-snapshots > MyComponent Matches snapshot when passed all props: when passed all props 1`] = `
<div>
  <span>
    Hello
  </span>
  jest-auto-snapshots String Fixture
  <NodeFixture />
</div>
`;

exports[`jest-auto-snapshots > MyComponent Matches snapshot when passed only required props: when passed only required props 1`] = `
<div>
  jest-auto-snapshots String Fixture
</div>
`;
